README.md

```diff
@@ -40,13 +39,12 @@ jobs:
       #
       # ... build & publish the artifact
       #
-      - name: Publish build artifact metadata
-        uses: babbel/publish-build-metadata@v1
+      - uses: babbel/publish-build-metadata@v1
         with:
-          access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          access_key_id:     ${{ secrets.AWS_ACCESS_KEY_ID }}
           secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          meta_table_arn: ${{ secrets.AWS_META_TABLE_ARN }}
-          slices: 'api, consumer.kinesis, consumer.firehose'
+          meta_table_arn:    ${{ secrets.AWS_META_TABLE_ARN }}
+          slices:            'api, consumer.kinesis, consumer.firehose'
```

Comment:
Why not just an array?

Reply:
I was trying at the beginning but it was producing error, actions don't support array attributes apparently - or at least I don't know how to do it xD

Reply:
~tabs vs spaces~ I'm not sure if I'm right but it looks like nothing is enforcing this kind of alignment of values. Do we really need this? It seems to be IDE/editor-specific configuration that requires extra steps to reproduce.